ratatool-extras/src/main/scala/com/spotify/ratatool/samplers/ParquetSampler.scala

```diff
@@ -30,7 +29,7 @@ import scala.collection.mutable.ListBuffer
  * Records are represented as Avro [[GenericRecord]].
  * Only head mode is supported.
  */
-class ParquetSampler(path: Path, protected val seed: Option[Long] = None)
+class ParquetSampler(path: String, protected val seed: Option[Long] = None)
```

Comment:
this might need to be changed back, per readme example. or the readme example needs to be changed, either one

Reply:
oops this is correct I will fix the readme